services/TS29222_CAPIF_Auditing_API/logs/core/auditoperations.py

```diff
@@ -0,0 +1,91 @@
+import sys
+
+from flask import current_app, Flask, Response
+import json
+
+from ..db.db import MongoDatabse
+from ..encoder import JSONEncoder
+from bson import json_util
+from ..models.protocol import Protocol
+
+
+class AuditOperations:
+
+    def __init__(self):
```

Comment:
The connection to the database is not made from the class but from an abstract class and we make the class that performs the operations on the database inherit from it (Review code)